Add RN Source Context to feature page and wizard (#7217)
getsentry/sentry-docs: src/wizard/react-native/index.md

---
name: React-Native

> doc_link: /platforms/react-native/

Author: Yep, removed it as you were writing the comment. :D

```

For more information, please refer to the [Sentry React Native documentation.](https://docs.sentry.io/platforms/react-native/performance/instrumentation/)

## Debug Symbols

We offer a range of methods to provide Sentry with debug symbols so that you can see symbolicated stack traces and triage issues faster.

> Complete stack traces will be shown for your React Native Javascript error by default using the [automatic source maps upload](/platforms/react-native/sourcemaps). For manual source maps upload follow [this guide](/platforms/react-native/sourcemaps).

Reviewer: I think we need to use absolute links in the wizard.

```suggestion
Complete stack traces will be shown for React Native Javascript errors by default using Sentry's [automatic source maps upload](https://docs.sentry.io/platforms/react-native/sourcemaps/). To set up manual source maps upload follow [this guide](https://docs.sentry.io/platforms/react-native/sourcemaps/).
```

Reviewer: Also, these links point to the same spot - is that intentional?

> Complete stack traces will be shown for React Native Javascript errors by default using Sentry's [automatic source maps upload](https://docs.sentry.io/platforms/react-native/sourcemaps/). To set up manual source maps upload follow [this guide](https://docs.sentry.io/platforms/react-native/sourcemaps/).

Author: Thanks for the catch. Done.

> You'll also need to [upload the debug information files](/platforms/react-native/upload-debug/) generated by the native iOS and Android tooling for native crashes.

Reviewer: ```suggestion
You'll also need to upload [Debug Symbols](https://docs.sentry.io/platforms/react-native/upload-debug/) generated by the native iOS and Android tooling for native crashes.
```

## Source Context

If Sentry has access to your application's source code, it can show snippets of code (_source context_) around the location of stack frames, which helps to quickly pinpoint problematic code.

> Source Context will be shown for React Native Javascript error by default if source maps are uploaded. For source maps upload follow [this guide](/platforms/react-native/sourcemaps).

Reviewer: ```suggestion
Source Context will be shown for React Native Javascript error by default if source maps are uploaded. To set up source maps upload, follow the [Source Maps](https://docs.sentry.io/platforms/react-native/sourcemaps/) guide.
```

> To enable source context for native errors, you'll need to upload native debug symbols to Sentry by following the [Uploading Source Code Context With Sentry Gradle Plugin](/platforms/react-native/upload-debug/#uploading-source-context-with-sentry-gradle-plugin) guide and [Uploading Source Context With Xcode](/platforms/react-native/upload-debug/#uploading-source-context-with-xcode).

Reviewer: ```suggestion
To enable source context for native errors, you'll need to upload native debug symbols to Sentry by following the instructions at [Uploading Source Code Context With Sentry Gradle Plugin](https://docs.sentry.io/platforms/react-native/upload-debug/#uploading-source-context-with-sentry-gradle-plugin) and [Uploading Source Context With Xcode](https://docs.sentry.io/platforms/react-native/upload-debug/#uploading-source-context-with-xcode).
```
